### PR TITLE
[ORT][TRT] support FaceFusion

### PR DESCRIPTION
**Pull request by wangzijian1010**
This pull request introduces new features and updates to the face recognition and manipulation functionalities in the project. It includes the addition of several new face-related models and their corresponding test files, as well as updates to the CMake configuration for these new models.

### New Features:

* Added new face-related models and their corresponding headers and source files:
  * `face_68landmarks` (`lite/ort/cv/face_68landmarks.h`, `lite/ort/cv/face_68landmarks.cpp`) [[1]](diffhunk://#diff-41329e60b479f56c850e286008c750d2975c5763d3ac037096b250d9176f5481R1-R45) [[2]](diffhunk://#diff-4e0079bdce93841dcf97019e9648e8fd2c5bda2021abd95a8ed1795c0c3cba79R1-R97)
  * `face_recognizer` (`lite/ort/cv/face_recognizer.h`, `lite/ort/cv/face_recognizer.cpp`)
  * `face_restoration` (`lite/ort/cv/face_restoration.h`, `lite/ort/cv/face_restoration.cpp`)
  * `face_swap` (`lite/ort/cv/face_swap.h`, `lite/ort/cv/face_swap.cpp`)
  * `face_fusion_pipeline` (`lite/ort/cv/face_fusion_pipeline.h`, `lite/ort/cv/face_fusion_pipeline.cpp`) [[1]](diffhunk://#diff-aff80075392dc8a6bce0f6cdd5bde12a288721441388c4a99ff35d31f19e59d6R1-R45) [[2]](diffhunk://#diff-f3d66a339e6b735d0cc5253b23465223401550623e1c400bf65aae0a4a773437R1-R38)

### Updates to CMake Configuration:

* Updated the `OpenCV_LIBS` variable to include `opencv_calib3d` in `cmake/opencv.cmake`.
* Added new executables for the new face-related models in `examples/lite/CMakeLists.txt`.

### Test Files:

* Added test files for the new face-related models:
  * `test_lite_face_68landmarks.cpp`
  * `test_lite_face_recognizer.cpp`
  * `test_lite_face_restoration.cpp`
  * `test_lite_face_swap.cpp`
  * `test_lite_facefusion_pipeline.cpp`

### Header File Updates:

* Updated `lite/models.h` to include the new face-related models and their typedefs. [[1]](diffhunk://#diff-80805648b47990fbf2c14ee25216641075e717ce1157a36d0089ab5108b89112R118-R122) [[2]](diffhunk://#diff-80805648b47990fbf2c14ee25216641075e717ce1157a36d0089ab5108b89112R510-R514) [[3]](diffhunk://#diff-80805648b47990fbf2c14ee25216641075e717ce1157a36d0089ab5108b89112R593-R611) [[4]](diffhunk://#diff-80805648b47990fbf2c14ee25216641075e717ce1157a36d0089ab5108b89112L621-R651)

### CODE REVIEW

File: lite/models.h [hunk 1]
```diff
@@ -580,6 +590,25 @@ namespace lite
...
         }
 
+        namespace swap
+        {
+            namespace facefusion
+            {
+                typedef _ONNXYOLOFaceNet YOLOV8Face;
+                typedef _ONNXFace_Swap InSwapper;
+                typedef _ONNXFace_Restoration GFPGAN;
+                typedef _ONNXFace_68Landmarks Face_68Landmarks;
```
Comment (DefTruth) [inline lite/models.h:600]: 这几个新增了类都属于FaceFusion系列，建议改成FaceFusion68Landmarks, FaceFusionRecognizer, FaceFusionSwap, FaceFusionRestoration, FaceFusionPipeline的风格。本仓库class名称采用驼峰法风格，不采用下划线。

**Review by DefTruth — CHANGES_REQUESTED** [review]

**Review by DefTruth — APPROVED** [review]
LGTM

File: lite/ort/cv/face_swap.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    std::tie(preprocessed_mat, affine_martix) = face_utils::warp_face_by_face_landmark_5(target_face,target_landmark_5,face_utils::ARCFACE_128_V2);
+
+    std::vector<float> crop_size= {128.0,128.0};
+    crop_list.emplace_back(face_utils::create_static_box_mask(crop_size));
+
+    cv::cvtColor(preprocessed_mat,preprocessed_mat,cv::COLOR_BGR2RGB);
+    preprocessed_mat.convertTo(preprocessed_mat,CV_32FC3,1.0 / 255.f);
+    preprocessed_mat.convertTo(preprocessed_mat,CV_32FC3,1.0 / 1.f,0);
+
+    std::vector<float> model_martix = face_utils::load_npy("/home/facefusion-onnxrun/python/model_matrix.npy");
```
Comment (DefTruth) [inline lite/ort/cv/face_swap.cpp:22]: 这里写了某个绝对路径，是否要修改？

Comment (wangzijian1010) [reply lite/ort/cv/face_swap.cpp:22]: 我现在在测试大佬  我准备做完多线程的之后 再一次性全部修复 先不 merge

Comment (DefTruth) [reply lite/ort/cv/face_swap.cpp:22]: 好的

**Comment by Arrooy** [comment]
Hello! Seems like this PR generates a problem. Fresh compilation produces:
fatal error: NvInfer.h: No such file or directory
    8 | #include "NvInfer.h"

**Pull request merged**